--- ./qtbase/src/corelib/io/qfilesystemengine_unix.cpp.org	2025-08-13 13:49:23.985721145 +0800
+++ ./qtbase/src/corelib/io/qfilesystemengine_unix.cpp	2025-08-13 13:50:31.959241294 +0800
@@ -100,7 +100,7 @@
 #  define FICLONE       _IOW(0x94, 9, int)
 #endif
 #endif
-
+#  undef STATX_BASIC_STATS
 #if defined(Q_OS_ANDROID)
 // statx() is disabled on Android because quite a few systems
 // come with sandboxes that kill applications that make system calls outside a
